tests/network/hotplug.go

```diff
@@ -101,6 +101,21 @@ var _ = SIGDescribe("nic-hotplug", func() {
 			waitForSingleHotPlugIfaceOnVMISpec(hotPluggedVMI)
 			hotPluggedVMI = verifyDynamicInterfaceChange(hotPluggedVMI, plugMethod)
 			Expect(libnet.InterfaceExists(hotPluggedVMI, vmIfaceName)).To(Succeed())
+
+			updatedVM, err := kubevirt.Client().VirtualMachine(hotPluggedVM.Namespace).Get(context.Background(), hotPluggedVM.Name, &metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			vmIfaceSpec := vmispec.LookupInterfaceByName(updatedVM.Spec.Template.Spec.Domain.Devices.Interfaces, ifaceName)
+			Expect(vmIfaceSpec).NotTo(BeNil(), "VM spec should contain the new interface")
```

Comment:
I think you also need to check that the mac is not empty.

Reply:
If KMP is not in the cluster, this check is irrelevant and we should not run it at all.

But, if KMP is in place, and the mac is not set on the VM, we have a problem or assuming all is good when it is not. Moreover, if we are not running the KMP, this will give a wrong impression that we test it.

Therefore, please elaborate in which case KMP is not expected to be running.
Is it expected to run on CI with kubevirtci?

Reply:
> Outside kubevirt CI it may fail in case KMP not exist in the env.

You can label the test with `kmp` or something like that, and if someone does not want to run this test it can opt-out by negating the label.

We have tests with Multus, why is that ok to depend on it in the e2e tests and not kmp?
And in practice, we do not depend on KMP, we depend on the fact that a mac is set there somehow (it can be any other webhook).

The test, as it is presented now, cannot be trusted to check that the change in this PR works.
I do not like it, but I will not block the PR on this one.

Reply:
> The test, as it is presented now, cannot be trusted to check that the change in this PR works.

Since we do have KMP on Kubevirt CI, the VM will get MAC address and the test actually verify the VMI got the MAC address as well.
What do see that is missing?

Reply:
You are assuming KMP is working, if it is not, it checks nothing.
This is why a check is needed, to assure the check is relevant.